birdhouse/config/phoenix/config/proxy/canarie_api_monitoring.py.template

```diff
@@ -0,0 +1,5 @@
+SERVICES['node']['monitoring']['Phoenix'] = {
+    'request': {
+        'url': 'https://${PAVICS_FQDN_PUBLIC}/geoserver/web/'
```

Comment:
I guess so. I was just copying from the old canarie-api config. We should probably make this configurable in case geoserver isn't available. 

Is there a better route for this actually? Something that checks if phoenix itself is running (as opposed to a service run through phoenix)?

Also, I think that phoenix was on the list of services to remove so maybe this isn't worth fixing right now.

Reply:
Yes Phoenix probably on the chopping list, pending PCIC confirmation since Ouranos and CRIM are not using it.

Reply:
`'http://${PAVICS_FQDN}:8443/'` should be used for monitoring.
There's no HTTPS nginx redirection with something like `/phoenix/` for the stats. 
Something was plugged to fill the schema that expects something there, but the correct fix is to have `phoenix` component extend the proxy with that route.